src/app/SharpRaven/Data/SentryStacktrace.cs

```diff
@@ -91,7 +84,7 @@ public override string ToString()
 
             StringBuilder sb = new StringBuilder();
 
-            foreach (var frame in Frames)
+            foreach (var frame in Frames.Reverse())
```

Comment:
Depends on how we want the `ToString` implementation to behave. There is a unit test making sure it returns a string being equivalent to the `Exception.ToString()`, and in order for that to work we have to reverse the frames here. 

(Though, I'm not sure if that unit test adds any direct value, and also it doesn't work on computers with other language settings than English. Does for example not work on my computer (Swedish).).